platform/src/main/java/org/hillview/utils/TestTables.java

```diff
@@ -18,6 +18,7 @@
 
 package org.hillview.utils;
 
+import com.sun.tools.javac.util.Assert;
```

Comment:
Because they're in the `test` folder, and `TestTables` is (contrary to what the name suggests) not in the `test` folder.